assets/src/edit-story/components/panels/animation/directionRadioInput.js

```diff
@@ -197,11 +197,7 @@ const translations = {
   [ROTATION.COUNTER_CLOCKWISE]: __('counterclockwise', 'web-stories'),
 };
 
-export const DirectionRadioInput = ({
-  directions = [],
-  onChange,
-  defaultChecked,
-}) => {
+export const DirectionRadioInput = ({ value, directions = [], onChange }) => {
```

Comment:
noticed a bug in this where using effect choooser to change direction could cause this input to get out of sync. 

Made this a controlled component to accommodate for that.